### ISSUE TITLE
Gmail account specially created appears offline in Google Talk after a moment

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#349]
```
What steps will reproduce the problem?
1. I use a gmail account specially created for the phone
2. I launch manually the connection in GTalkSMS : the new user appears online 
in my Google Talk (from the Gmail page).
4. After a moment, the phone fall in standby.
5. After an other moment, the new user appears offline in my Google Talk but 
online ine the app.
6. If i wake up the phone, nothing changes.

What is the expected output?

That the new account stays online.


Thanks for all.

Additional informations:
Version : GTalkSMS 4.4.4
Device : Nexus 4
Android : 4.4

```

Original issue reported on code.google.com by `fabrice....@gmail.com` on 16 Jan 2014 at 7:48

Attachments:
- [log.txt](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-349/comment-0/log.txt)

**Comment by GoogleCodeExporter** [comment]
```
If i wake up the phone from standby while it's charging, the contact comes back 
online after few seconds.
```

Original comment by `fabrice....@gmail.com` on 17 Jan 2014 at 11:38

**Comment by GoogleCodeExporter** [comment]
```
Might be fixed with the last version of asmack (internal lib) in the 4.4.5 
version (beta stage for now).
Please reopen if not.
```

Original comment by `Florent....@gmail.com` on 16 Aug 2014 at 11:44
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
```
Ok thank you, i'll try
```

Original comment by `fabrice....@gmail.com` on 24 Aug 2014 at 12:40

**Comment by GoogleCodeExporter** [comment]
```
Released on Play Market as 5.0
```

Original comment by `Florent....@gmail.com` on 25 Oct 2014 at 6:23
- Changed state: **Deployed**
